pkg/controller/unifiedpushserver/unifiedpushserver_controller.go

```diff
@@ -281,30 +275,6 @@ func (r *ReconcileUnifiedPushServer) Reconcile(request reconcile.Request) (recon
 		return r.manageError(instance, err)
 	}
 
-	// look for other unifiedPush resources and don't provision a new one if there is another one with Phase=Complete
-	existingInstances := &pushv1alpha1.UnifiedPushServerList{
-		TypeMeta: metav1.TypeMeta{
-			Kind:       "UnifiedPushServer",
-			APIVersion: "push.aerogear.org/v1alpha1",
-		},
-	}
-	opts := &client.ListOptions{Namespace: instance.Namespace}
```

Comment:
This breaks like 947 which references opts which hasn't been defined.

Reply:
Argh, I obviously tried this out _before_ making this last minute change :roll_eyes: 

Fixed, thanks.